yt_dlp/extractor/canvas.py

```diff
@@ -40,10 +41,9 @@ class CanvasIE(InfoExtractor):
     }]
     _GEO_BYPASS = False
     _HLS_ENTRY_PROTOCOLS_MAP = {
-        'HLS': 'm3u8_native',
```

Comment:
Are you sure HLS always have DRM? Otherwise, it shouldn't be removed like this

Reply:
I re-added the dict with `HLS` / `HLS_AES` for easier switching in case of issues. Both use `m3u8_native`

Tested a few cases, for example 
```
python3 -m yt_dlp https://www.vrt.be/vrtnu/a-z/thuis/27/thuis-s27a5090/ --netrc -Uv 
python3 -m yt_dlp https://www.vrt.be/vrtnu/a-z/sabena/1/sabena-s1a1-het-grote-avontuur/ --netrc -Uv 
```